Error: A checkbox group must have an accessible label. Use `fieldset` and `legend`. Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`. Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group/3/e51086ff-2ea9-4f9d-b35f-6c58fb76479d.html b/output/Checkbox Group/3/e51086ff-2ea9-4f9d-b35f-6c58fb76479d.html
index 06a0dc4..09b24a1 100644
--- a/output/Checkbox Group/3/e51086ff-2ea9-4f9d-b35f-6c58fb76479d.html	
+++ b/output/Checkbox Group/3/e51086ff-2ea9-4f9d-b35f-6c58fb76479d.html	
@@ -1,3 +1,15 @@
-<label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-<label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-<label><input type="checkbox" name="fruits" value="apples"> Apples</label>
\ No newline at end of file
+<fieldset>
+  <legend>What fruits do you like?</legend>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
+</fieldset>
\ No newline at end of file
